Contained are the code and data needed to re-create the analyses and figures in: <h4><a href="http://rspb.royalsocietypublishing.org/content/284/1856/20170914
" target="_blank"><font color = "#b2182b">Des Marais et al. (in press)</font></a></h4>  If you use any of the code, figures, analyses, or novel ideas, please cite the article below-- there's a<a href="http://biorxiv.org/content/early/2016/10/30/080804" target="_blank"><font color = "#b2182b">pre-print</font></a> as well if you get stuck outside a paywall.  If you use any of the data, please cite the appropriate manuscript(s).

*Citation* 

Des Marais DL, Guerrero RF, Lasky JR, Scarpino SV. in press. Topological features of a gene co-expression network predict patterns of natural diversity in environmental response. Proceedings of the Royal Society B.

*Preprint* 

Des Marais, D. L., Guerrero, R., Lasky, J. R., & Scarpino, S. V. (2016). Topological features of gene regulatory networks predict patterns of natural diversity in environmental response. bioRxiv, 080804.


*Data*

[Cubillos ASE](http://www.plantcell.org/content/26/11/4298.short)
Cubillos, F. A., Stegle, O., Grondin, C., Canut, M., TisnÃ©, S., Gy, I., & Loudet, O. (2014). Extensive cis-regulatory variation robust to environmental perturbation in Arabidopsis. The Plant Cell, 26(11), 4298-4310.

[DaveAllGxTDrought](http://www.plantcell.org/content/24/3/893.full)
Des Marais, D. L., McKay, J. K., Richards, J. H., Sen, S., Wayne, T., & Juenger, T. E. (2012). Physiological genomics of response to soil drying in diverse Arabidopsis accessions. The Plant Cell, 24(3), 893-914.

[DaveAllTDrought](http://www.plantcell.org/content/24/3/893.full)
Des Marais, D. L., McKay, J. K., Richards, J. H., Sen, S., Wayne, T., & Juenger, T. E. (2012). Physiological genomics of response to soil drying in diverse Arabidopsis accessions. The Plant Cell, 24(3), 893-914.

[FDR0.1](http://www.plantcell.org/content/24/3/893.full)
Des Marais, D. L., McKay, J. K., Richards, J. H., Sen, S., Wayne, T., & Juenger, T. E. (2012). Physiological genomics of response to soil drying in diverse Arabidopsis accessions. The Plant Cell, 24(3), 893-914.

[G&E](http://www.plantcell.org/content/24/3/893.full)
Des Marais, D. L., McKay, J. K., Richards, J. H., Sen, S., Wayne, T., & Juenger, T. E. (2012). Physiological genomics of response to soil drying in diverse Arabidopsis accessions. The Plant Cell, 24(3), 893-914.

[Splitting Drought](http://www.plantcell.org/content/24/3/893.full)
Des Marais, D. L., McKay, J. K., Richards, J. H., Sen, S., Wayne, T., & Juenger, T. E. (2012). Physiological genomics of response to soil drying in diverse Arabidopsis accessions. The Plant Cell, 24(3), 893-914.

[Expression Level](https://www.ncbi.nlm.nih.gov/pubmed/21389272)
Yang, L., & Gaut, B. S. (2011). Factors that contribute to variation in evolutionary rate among Arabidopsis genes. Molecular biology and evolution, 28(8), 2359-2369.

[Feltus_etal_AdditionalFile3_EdgeLists](http://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-7-44)
Feltus, F. A., Ficklin, S. P., Gibson, S. M., & Smith, M. C. (2013). Maximizing capture of gene co-expression relationships through pre-clustering of input expression samples: an Arabidopsis case study. BMC systems biology, 7(1), 44.

[gene_expression_net_feltus_main](http://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-7-44)
Feltus, F. A., Ficklin, S. P., Gibson, S. M., & Smith, M. C. (2013). Maximizing capture of gene co-expression relationships through pre-clustering of input expression samples: an Arabidopsis case study. BMC systems biology, 7(1), 44.

[HannahAllGxTCold](http://www.plantphysiol.org/content/142/1/98)
Hannah, M. A., Wiese, D., Freund, S., Fiehn, O., Heyer, A. G., & Hincha, D. K. (2006). Natural genetic variation of freezing tolerance in Arabidopsis. Plant physiology, 142(1), 98-112.

[HannahAllTCold](http://www.plantphysiol.org/content/142/1/98)
Hannah, M. A., Wiese, D., Freund, S., Fiehn, O., Heyer, A. G., & Hincha, D. K. (2006). Natural genetic variation of freezing tolerance in Arabidopsis. Plant physiology, 142(1), 98-112.

[TPC2015_00910](http://www.plantcell.org/content/early/2016/02/03/tpc.15.00910.full.pdf)
Bechtold, U., Penfold, C. A., Jenkins, D. J., Legaie, R., Moore, J. D., Lawson, T., ... & Hickman, R. (2016). Time-series transcriptomics reveals that AGAMOUS-LIKE22 links primary metabolism to developmental processes in drought-stressed Arabidopsis. The Plant Cell, TPC2015-00910.

[comm_props_raw_21Jan16](http://biorxiv.org/content/early/2016/10/30/080804)
Des Marais DL, Guerrero RF, Lasky JR, Scarpino SV. in press. Topological features of a gene co-expression network predict patterns of natural diversity in environmental response. Proceedings of the Royal Society B.

*Legal*
 
The data aren't all ours, but we did write a lot of the code.  Obviously everything, e.g., code, data, etc., are provided without any guarantee about accuracy, quality, or the impact factor of the journal you can publish them in.  Please cite the appropriate paper for each data set and the appropriate paper for any internal code (e.g., if you use any R libraries). 

*License* 

GPL (>= 3.0), where applicable.